src/components/AudioTrack/PlayPause.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <button
     type="button"
-    class="flex items-center justify-center bg-dark-charcoal h-20 w-20 rounded-sm transition-shadow duration-100 ease-linear disabled:opacity-70 focus:outline-none focus-visible:ring focus-visible:ring-offset-2 focus-visible:ring-pink"
-    @click="toggle"
+    class="play-pause flex items-center justify-center bg-dark-charcoal transition-shadow duration-100 ease-linear disabled:opacity-70 focus:outline-none focus-visible:ring focus-visible:ring-offset-2 focus-visible:ring-pink"
```

Comment:
Can `@apply` be used here?

Reply:
I'm sorry but I didn't understand what you meant.

Reply:
She's referring to using a class here, like ".play-button", and putting all these tailwind classes in an @apply in the style block of the component.

Which in general we need to decide if that's a pattern we want to embrace or not.

Reply:
Using `@apply` feels fundamentally wrong **to me** because I feel like Tailwind's full potential can be utilised when you don't have to write any CSS at all. I only use @apply for situations where 
- the same classes are being repeated in many places
- the classes (if combined) make a meaningful "micro-component"

but those are just my thoughts. We should definitely discuss this in the chat.